src/index.js

```diff
@@ -836,71 +948,73 @@ export default function (babel) {
         let idx = path.node.idx || path.scope.generateUidIdentifier("i");
         let len = path.scope.generateUidIdentifier("len");
         let declarations = [
-          t.variableDeclarator(idx, t.numericLiteral(0)),
-          t.variableDeclarator(
+          nb("variableDeclarator", idx, nb("numericLiteral", 0)),
```

Comment:
I doubt that's correct. `path.node.idx` might point, say, to some identifier that may potentially be way up at the top of the file. If we make declarations inherit from idx, then we're saying "this variable declarations belongs way up at the top of the file.